benchmarks/operator_benchmark/pt/qembeddingbag_test.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import operator_benchmark as op_bench
+import torch
+import torch.nn.quantized.dynamic as nnqd
+import numpy
+from . import configs
+
+"""
+Microbenchmarks for qEmbeddingBag operators.
+"""
+
+class QEmbeddingBagBenchmark(op_bench.TorchBenchmarkBase):
+    def init(self, embeddingbags, dim, mode, input_size, offset, sparse, include_last_offset, device):
+        self.embedding = nnqd.EmbeddingBag(
+            num_embeddings=embeddingbags,
+            embedding_dim=dim,
+            mode=mode,
+            include_last_offset=include_last_offset).to(device=device)
+        numpy.random.seed((1 << 32) - 1)
```

Comment:
just curious, what's the context on this line?

Reply:
It was used in the embeddingbag_test. So I did the same here for consistency